From 14316e0214a2439504e6730fc3c454bd1792afaa Mon Sep 17 00:00:00 2001
From: Fabio Estevam <festevam@gmail.com>
Date: Fri, 12 Jan 2024 22:27:06 -0300
Subject: [PATCH] its: Add an entry for the asher board

Adapt the sama5d27_wlsom1_ek.its to work with the asher board.

Signed-off-by: Fabio Estevam <festevam@gmail.com>
---
 sama5d27_wlsom1_ek.its | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sama5d27_wlsom1_ek.its b/sama5d27_wlsom1_ek.its
index 08837137044e..fd573b2c1974 100644
--- a/sama5d27_wlsom1_ek.its
+++ b/sama5d27_wlsom1_ek.its
@@ -9,7 +9,7 @@
 /dts-v1/;
 
 / {
-	description = "Microchip SAMA5D27 WLSOM1 EK FIT Image";
+	description = "Microchip SAMA5D27 WLSOM1 Asher FIT Image";
 	#address-cells = <1>;
 
 	images {
@@ -32,7 +32,7 @@
 
 		base_fdt {
 			description = "Flattened Device Tree blob";
-			data = /incbin/("./at91-sama5d27_wlsom1_ek.dtb");
+			data = /incbin/("./at91-sama5d27_wlsom1_asher.dtb");
 			type = "flat_dt";
 			arch = "arm";
 			compression = "none";
-- 
2.34.1

